app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -45,6 +49,15 @@ public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
     @Override
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
+            case R.id.learn_more:
```

Comment:
@mcomella can you add a telemetry probe here? Would be nice to have to see if people are clicking it.